de.dlr.sc.virsat.target/tmodel-src/virsat.tmodel

```diff
@@ -1,6 +1,6 @@
- Target virsat {
+Target virsat {

```

Comment:
What do you think of a virsat_windows target? To be consistent?

Reply:
@SaMuellerDLR  what do you think about it?

Reply:
@Anna-Lena-DLR Sure 👍